doc/events.md

```diff
@@ -12,3 +12,7 @@
 
 * `INTERRUPT` - emitted on signal events `SIGHUP`, `SIGINT` or `SIGTERM`. The event is emitted with 1 argument `data`:
     * `data.exitCode` - exit code with which gemini will be interrupted
+
+* `START_BROWSER` - emitted on browser session start. Emitted with [browser instance](../lib/browser/new-browser.js). In case of handler return promise that promise will be waited before go on.
```

Comment:
If handler return**s** **a** promise tests will be executed in this session only after the promise is resolved.

??